### ISSUE TITLE
仓库更新报告 (2025-03-31)

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#12]
# 每日更新报告（2025-03-31）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-04-01 04:47:32 | Harry Mellor | Move dockerfiles into their own directory (#14549) |
| 2025-04-01 03:15:21 | Mark McLoughlin | [V1][Core] Remove unused speculative config from scheduler (#15818) |
| 2025-04-01 01:27:07 | Harry Mellor | Fix Transformers backend compatibility check (#15290) |
| 2025-04-01 01:25:20 | Alexander Matveev | [V1] TPU CI - Add basic perf regression test (#15414) |
| 2025-04-01 01:00:50 | yihong | fix: change GB to GiB in logging close #14979 (#15807) |
| 2025-04-01 00:19:35 | shangmingc | [V1][Spec Decode] Remove deprecated spec decode config params (#15466) |
| 2025-04-01 00:01:35 | Cyrus Leung | [Bugfix] Check dimensions of multimodal embeddings in V1 (#15816) |
| 2025-03-31 23:59:37 | Harry Mellor | Upgrade `transformers` to `v4.50.3` (#13905) |
| 2025-03-31 21:56:42 | Mrm | [Bugfix] Fix missing return value in load_weights method of adapters.py (#15542) |
| 2025-03-31 21:23:53 | Alex Brooks | [Bugfix] Fix Crashing When Loading Modules With Batchnorm Stats (#15813) |
| 2025-03-31 20:35:14 | Naveassaf | [Model] Update support for NemotronNAS models (#15008) |
| 2025-03-31 20:22:34 | youkaichao | [V1] Fully Transparent Implementation of CPU Offloading (#15354) |
| 2025-03-31 20:13:32 | yihong | fix: better install requirement for install in setup.py (#15796) |
| 2025-03-31 19:54:49 | Harry Mellor | Recommend developing with Python 3.12 in developer guide (#15811) |
| 2025-03-31 19:42:18 | Charlie Fu | [Feature][ROCm]Enable fusion pass for torch.compile on ROCm (#15050) |
| 2025-03-31 15:38:58 | Jennifer Zhao | [Benchmark] Update Vision Arena Dataset and HuggingFaceDataset Setup (#15748) |
| 2025-03-31 10:39:56 | Chengyang LIU | [Misc] update the comments (#15780) |

## sgl-project/sglang

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-04-01 07:14:49 | Yineng Zhang | use fa3 in sgl-kernel (#4954) |
| 2025-04-01 03:00:34 | Yuhong Guo | [Build] Fix cuda12.8 build error in nvfp4_scaled_mm_kernels.cu (#4953) |
| 2025-04-01 02:24:09 | Yineng Zhang | bump sgl-kernel v0.0.6 (#4950) |
| 2025-04-01 00:57:51 | Mick | refactor: multimodal data (#4754) |
| 2025-03-31 17:08:55 | yinfan98 | [Fix] revert clean m.def for cudagraph (#4944) |
| 2025-03-31 15:04:21 | JieXin Liang | [feat] interface for platforms abstraction (#4928) |
| 2025-03-31 12:34:21 | Zhiqiang Xie | Fix oom error for large page size (#4913) |
| 2025-03-31 11:46:07 | Baizhou Zhang | [Fix] Add torch compile for torch.clamp back (#4936) |
| 2025-03-31 11:10:21 | fzyzcjy | Fix DeepSeek bug causing 2.2% MMLU drop when TP!=DP (#4883) |
| 2025-03-31 10:40:14 | Baizhou Zhang | [Fix] Improve Lora tests and reduce CI runtime (#4925) |
